sync.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+cwd=$(pwd)
+if [ ! -f sync ]; then
+    go get -u -f github.com/dkonovenschi/aggretastic-sync
```

Comment:
aggretastic-sync can be just a subpackage of aggretastic. No reason to handle a separate package - as it's not a generic thing, it's used for syncing only this aggretastic repo.

So, you can just include it in this MR too